modules/ROOT/pages/dataweave-cookbook-defaults.adoc

```diff
@@ -65,23 +69,45 @@ and
 xref:dataweave-flow-control.adoc#control_flow_else_if[else-if]
 statements to specify a default value.
 
-The following example uses the string `"value is even"` as its default.
+The following example sets `US` as the default value for `userLocation` if it is not present in the JSON input message:
 
-.DataWeave Script:
 [source,DataWeave, linenums]
 ----
 %dw 2.0
-var myVar = (1 + 1)
 output application/json
 ---
-if (isOdd(myVar)) "value is odd"
-else "value is even"
+if (payload.location != null) {
+	"userLocation" : payload.location
+} else {
+	"userLocation" : "United States"
+}
 ----
 
-.Output:
+* If the Mule Application receives the following JSON input:
++
+[source,json, linenums]
+----
+{
+  "location": "Argentina"
+}
+----
++
+The DataWeave transformation output is:
++
 [source,JSON,linenums]
 ----
-"value is even"
+{
+  "userLocation": "Argentina"
+}
+----
+
+* If the Mule Application receives an empty JSON message, the DataWeave transformation output is:
++
+[source,JSON,linenums]
+----
+{
+  "userLocation": "United States"
+}
 ----
 
 == Setting a Default When Matching Patterns
```

Comment:
Please apply suggestions on previous two subtopics to this one as well:
title
method description
typical behavior
atypical values missing and method solution

Reply:
Remainder copyedited only